### PR TITLE
Add fs operations GUI

### LINKED ISSUE #14: Add FS operations GUI
- [ ] Add ability to upload new files
 - [ ] Add ability to create folders 
 - [ ] Add ability to delete existing files/folders
 - [ ] Add ability to rename files/folders

### PR DESCRIPTION
**Pull request by HeDo88TH**
Closes #14

### CODE REVIEW

File: webapp/js/components/ViewDataset.vue [hunk 1]
```diff
@@ -33,7 +34,51 @@
         </TabSwitcher>
         <Properties v-if="showProperties" :files="selectedFiles" @onClose="handleCloseProperties" />
     </Panel>
-</div>
+
+    <sui-modal v-model="uploadDialogOpen">
```
Comment (pierotofy) [inline webapp/js/components/ViewDataset.vue:38]: We have a `Window` class for modals. Did you check it out?

File: webapp/js/main.js [hunk 2]
```diff
@@ -10,6 +11,7 @@ import ViewDataset from './components/ViewDataset.vue';
 import Upload from './components/Upload.vue';
 import reg from './libs/sharedRegistry';
 import { setTitle, queryParams, inIframe } from './libs/utils';
+import SuiVue from 'semantic-ui-vue';
```
Comment (pierotofy) [inline webapp/js/main.js:14]: You *probably* don't need this one.

File: webapp/js/components/ViewDataset.vue (hunk 1 above)
Comment (pierotofy) [reply webapp/js/components/ViewDataset.vue:38]: https://github.com/DroneDB/CommonUI/blob/master/components/Window.vue#L33

**Comment by pierotofy** [comment]
A few comments:

 - When pressing the `Create new folder` button, the focus should go automatically to the input text field (otherwise I have to click it).
 - Pressing "Upload" should probably just open the dialog to upload files (not open a dialog for me to click to then upload files). It seems unnecessary to click Upload to then click Browse?

**Comment by HeDo88TH** [comment]
Closes #20

**Comment by HeDo88TH** [comment]
@pierotofy can we merge it? 🚀

**Pull request merged**